k2/csrc/util/logging.cc

```diff
@@ -0,0 +1,55 @@
+// k2/csrc/util/logging.h
+
+// Copyright (c) 2020 Xiaomi Corporation ( authors: Meixu Song )
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/csrc/util/logging.h"
+
+#ifdef K2_USE_GLOG
+// Google glog's api does not have an external function that allows one to check
+// if glog is initialized or not. It does have an internal function - so we are
+// declaring it here. This is a hack but has been used by a bunch of others too.
+// (e.g. Torch).
```

Comment:
it's better if you add the corresponding code link of `PyTorch`? (and I don't know what `Torch` here is, sorry)